.env.template

```diff
@@ -82,8 +82,13 @@ OPENAI_API_KEY=your-openai-api-key
 
 ## SMART_LLM_MODEL - Smart language model (Default: gpt-4)
 ## FAST_LLM_MODEL - Fast language model (Default: gpt-3.5-turbo)
+## RISK_EVALUATION_MODEL - Risk evaluation model (Default: gpt-4)
 # SMART_LLM_MODEL=gpt-4
 # FAST_LLM_MODEL=gpt-3.5-turbo
+# RISK_EVALUATION_MODEL=gpt-4
```

Comment:
Forgive me if this has been asked before, but why not use the smart model for this instead of having a separate setting for a risk eval model?

Reply:
I am asking whether using the FAST_LLM_MODEL for risk eval will generally have the feature work as intended.

Having configuration parameters with overlapping purpose or meaning always increases the maintenance burden. So if we can avoid adding one, that would be nice.

Reply:
Well, I'd advise against that - gpt-3.5 can work well enough, but is generally stupider (in the sense that it has a harder time catching nuance, for example when evaluating shell commands) and more prone to extremes.

Reply:
Alright, thanks for elaborating!